app/uk/gov/hmrc/excisemovementcontrolsystemapi/filters/MovementFilter.scala

```diff
@@ -75,6 +84,7 @@ object MovementFilter {
             case "ern" => FilterErn(o._2)
             case "lrn" => FilterLrn(o._2)
             case "arc" => FilterArc(o._2)
+            case "updatedSince" => FilterUpdatedSince(o._2.map(Instant.parse(_)))
```

Comment:
What happens if the Instant isn't parsed properly, I can't see the error handling code, is this converted to an Internal Server Error or is the Exception shown to the user? If it fails to parse we should probably return a BadRequest with our standard Error Response?